Makefile

```diff
@@ -95,8 +95,16 @@ release: all deps docs $(SMF_MANIFESTS)
 	$(TOP)/smf \
 	$(TOP)/npm \
 	$(RELSTAGEDIR)/$(NAME)
+	rm -rf $(RELSTAGEDIR)/$(NAME)/node_modules/eslint \
```

Comment:
@misterdjules I thought about doing that, but we'd probably still want this `rm -rf`in case you'd manually already run `npm install --dev`, right? I guess alternatively we could create a manifest of which modules we *do* want to include based on package.json?